runtime/gc_ctrl.c

```diff
@@ -787,3 +788,15 @@ CAMLprim value caml_ml_runtime_warnings_enabled(value unit)
   CAMLassert (unit == Val_unit);
   return Val_bool(caml_runtime_warnings);
 }
+
+CAMLprim value caml_ml_enable_float_array_alloc_warning(value vsize)
+{
+  caml_float_array_alloc_warning = Int_val(vsize);
+  return Val_unit;
+}
+
+CAMLprim value caml_ml_float_array_alloc_warning_enabled(value unit)
+{
+  CAMLassert (unit == Val_unit);
+  return Val_bool(caml_float_array_alloc_warning);
```

Comment:
This seems inconsistent with the use of `Int_val ` above.

Reply:
Fixed, thanks!